Episode 0: 0
/home/navin/projects/M2P/Re_M2P/utils_env.py:223: RuntimeWarning: invalid value encountered in divide
  all_feature_matrix = (all_feature_matrix-mean)/std
> /home/navin/projects/M2P/Re_M2P/cluttered_v7.py(178)step()
-> state= global_features - object_features
> /home/navin/projects/M2P/Re_M2P/cluttered_v7.py(181)step()
-> state = pad_state(state)
> /home/navin/projects/M2P/Re_M2P/cluttered_v7.py(184)step()
-> return state, reward, action==self.target or len(self.object_ids)<10, len(self.object_ids)
/home/navin/projects/M2P/Re_M2P/utils_env.py:32: RuntimeWarning: invalid value encountered in divide
  normalized_features[obj_id] = (np.array(feature_list) - min_vals) / range_vals
/home/navin/projects/M2P/Re_M2P/utils_env.py:223: RuntimeWarning: invalid value encountered in divide
  all_feature_matrix = (all_feature_matrix-mean)/std
> /home/navin/projects/M2P/Re_M2P/cluttered_v7.py(176)step()
-> if np.any(np.isnan(object_features)):
*** NameError: name 'cc' is not defined
> /home/navin/projects/M2P/Re_M2P/cluttered_v7.py(178)step()
-> state= global_features - object_features
> /home/navin/projects/M2P/Re_M2P/cluttered_v7.py(181)step()
-> state = pad_state(state)
> /home/navin/projects/M2P/Re_M2P/cluttered_v7.py(184)step()
-> return state, reward, action==self.target or len(self.object_ids)<10, len(self.object_ids)
Traceback (most recent call last):
  File "/home/navin/projects/M2P/Re_M2P/agent5_ddqn.py", line 201, in <module>
    episode_rewards = mini_batch_train(
  File "/home/navin/projects/M2P/Re_M2P/agent5_ddqn.py", line 160, in mini_batch_train
    next_state, reward, done, _ = env.step(action)
  File "/home/navin/projects/M2P/Re_M2P/cluttered_v7.py", line 168, in step
    partial, total = self._get_obs()
  File "/home/navin/projects/M2P/Re_M2P/cluttered_v7.py", line 93, in _get_obs
    self.partial_obs, self.full_obs = self.get_segmented_point_clouds()
  File "/home/navin/projects/M2P/Re_M2P/cluttered_v7.py", line 244, in get_segmented_point_clouds
    global_points = np.vstack(global_points)
  File "<__array_function__ internals>", line 200, in vstack
  File "/home/navin/.local/lib/python3.8/site-packages/numpy/core/shape_base.py", line 296, in vstack
    return _nx.concatenate(arrs, 0, dtype=dtype, casting=casting)
  File "<__array_function__ internals>", line 200, in concatenate
ValueError: need at least one array to concatenate
/usr/lib/python3/dist-packages/apport/report.py:13: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
  import fnmatch, glob, traceback, errno, sys, atexit, imp, stat
Traceback (most recent call last):
  File "/home/navin/projects/M2P/Re_M2P/agent5_ddqn.py", line 201, in <module>
    episode_rewards = mini_batch_train(
  File "/home/navin/projects/M2P/Re_M2P/agent5_ddqn.py", line 160, in mini_batch_train
    next_state, reward, done, _ = env.step(action)
  File "/home/navin/projects/M2P/Re_M2P/cluttered_v7.py", line 168, in step
    partial, total = self._get_obs()
  File "/home/navin/projects/M2P/Re_M2P/cluttered_v7.py", line 93, in _get_obs
    self.partial_obs, self.full_obs = self.get_segmented_point_clouds()
  File "/home/navin/projects/M2P/Re_M2P/cluttered_v7.py", line 244, in get_segmented_point_clouds
    global_points = np.vstack(global_points)
  File "<__array_function__ internals>", line 200, in vstack
  File "/home/navin/.local/lib/python3.8/site-packages/numpy/core/shape_base.py", line 296, in vstack
    return _nx.concatenate(arrs, 0, dtype=dtype, casting=casting)
  File "<__array_function__ internals>", line 200, in concatenate
ValueError: need at least one array to concatenate
